libsolidity/interface/AssemblyStack.h

```diff
@@ -71,6 +71,8 @@ class AssemblyStack
 	/// Multiple calls overwrite the previous state.
 	bool analyze(assembly::Block const& _block, Scanner const* _scanner = nullptr);
 
+	void optimize();
```

Comment:
Should add a comment here. Extra detail to be mentioned: not valid on loose assembly.

Reply:
Added a comment.